KarhooUISDK/Screens/BookingRequestScreen/FormBookingRequestScreen/FormBookingRequestViewController.swift

```diff
@@ -51,25 +151,49 @@ final class FormBookingRequestViewController: UIViewController, BookingRequestVi
     }
     
     override func loadView() {
-        setUpView()
-    }
-    
-    private func setUpView() {
         view = UIView()
         view.translatesAutoresizingMaskIntoConstraints = false
-        
-        container = UIView()
-        container.translatesAutoresizingMaskIntoConstraints = false
-        container.accessibilityIdentifier = "container_view"
-        container.backgroundColor = .white
         view.addSubview(container)
         
-        exitButton = UIButton(type: .custom)
-        exitButton.translatesAutoresizingMaskIntoConstraints = false
-        exitButton.accessibilityIdentifier = KHBookingRequestViewID.exitButton
-        exitButton.setImage(UIImage.uisdkImage("close_button"), for: .normal)
-        exitButton.addTarget(self, action: #selector(closePressed), for: .touchUpInside)
-        exitButton.imageView?.contentMode = .scaleAspectFit
+        bookingButton.set(actions: self)
+        termsConditionsView = TermsConditionsView()
+        
+        finishLoadingView()
+    }
+    
+    private func finishLoadingView() {
+        switch Karhoo.configuration.authenticationMethod() {
```

Comment:
I would suggest to use the same method setUpView for all auth methods, however you can hide the passengerdetailsview 

```
private func finishLoadingView() {
        setUpView()
        if presenter.isKarhooUser() {
            passengerDetailsView.isHidden = true
            passengerDetailsTitle.isHidden = true
            commentsInputText.isHidden = true
        }
    }
```

Reply:
this is updated as well